regulations/views/comment.py

```diff
@@ -43,9 +43,34 @@ def upload_proxy(request):
     })
 
 
+def download_proxy(request):
+    """Generate a temporary URL to GET an attachment."""
+    key = request.GET.get('key')
+    name = request.GET.get('name')
+    if key is None or name is None:
+        return JsonResponse(
+            {'message': 'Must provide key and name'},
+            status=400,
+        )
+    s3 = tasks.make_s3_client()
+    disposition = 'attachment; filename="{}"'.format(name)
+    url = s3.generate_presigned_url(
+        ClientMethod='get_object',
+        Params={
+            'ResponseContentDisposition': disposition,
+            'Bucket': settings.ATTACHMENT_BUCKET,
+            'Key': key,
+        },
+    )
+    return JsonResponse({'url': url})
```

Comment:
Sounds great. Let's do 90 days and we'll see if we can't tighten it to the comment deadline when that data's available.